docs/features/data_in_dynamodb.md

```diff
@@ -59,80 +60,24 @@ cumulus-api CLI also includes a backup and restore command. The CLI backup comma
 
 To backup a table with the CLI, make sure `@cumulus/api` package is installed. Then run:
 
-     $ ./node_modules/.bin/cumulus-api backup --table <table-name>
+```bash
```

Comment:
So people will still have to install the API package to do backup/restore, even though that package is no longer required for deploying Cumulus? Thus, they still need npm?

Reply:
They don't *have* to, but they *can*.

Reply:
Since node/npm are no longer in the main deployment docs, do we at least need a basic primer here or mention them as dependencies? Otherwise how to install `@cumulus/api` might not be clear

Reply:
Hmm, fair enough.   How about e29b4cf3d?

Reply:
Had a minor suggestion

Reply:
👍